### PR TITLE
Added advanced usage example for a notebook interacting with a Cloud …

### PR DESCRIPTION
**Pull request by nhira**
…TPU cluster

## Fixes / Features
- changed README.md to add reference to advanced usage (xpk-notebooks.md)
-

## Testing / Documentation
Testing details.

- [ y/n ] Tests pass
- [ y/n ] Appropriate changes to documentation are included in the PR

### CODE REVIEW

File: xpk-notebooks.md [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+```
+
+### XPK create cluster
+Install XPK.  (You know, this repo!)
+
+Create a GKE Cloud TPU cluster using XPK.
+```shell
+xpk cluster create --cluster ${CLUSTER} \
+--project=${PROJECTID} --default-pool-cpu-machine-type=n2-standard-8 \
+--num-slices=2 --tpu-type=v5litepod-256 --zone=${ZONE} \
```
Comment (yejingxin) [inline xpk-notebooks.md:56]: currently lws only support single slice, let us keep the num_slices=1

File: xpk-notebooks.md [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+          cloud.google.com/gke-tpu-topology: 16x16
+          cloud.google.com/gke-tpu-accelerator: tpu-v5-lite-podslice
+        tolerations:
+        - key: "google.com/tpu"
+          operator: "Exists"
+          effect: "NoSchedule"
+        containers:
+        - name: jupyter-notebook-server
+          image: jupyter/base-notebook:latest
+          # env:
```
Comment (yejingxin) [inline xpk-notebooks.md:197]: let us remove those comments

File: xpk-notebooks.md [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+          ports:
+          - containerPort: 8471 # Default port using which TPU VMs communicate
+          securityContext:
+            privileged: true
+          command:
+          - bash
+          - -c
+          - |
+            sleep 20
+            cat /app/ipp/security/ipcontroller-client.json
```
Comment (yejingxin) [inline xpk-notebooks.md:252]: also those two debug lines

**Review by yejingxin — COMMENTED** [review]
Looks good overall, added a few comments to clean up debug code.

**Review by Obliviour — APPROVED** [review]
Approving so that after @yejingxin approves, this can be merged.

**Pull request merged**